Examples/SPM/RichEditorViewSample/ViewController.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  ViewController.swift
+//  RichEditorViewSample
+//
+//  Created by Caesar Wirth on 4/5/15.
+//  Copyright (c) 2015 Caesar Wirth. All rights reserved.
+//
+
+import UIKit
+import RichEditorView
+import SafariServices
+
+class ViewController: UIViewController {
+    @IBOutlet var editorView: RichEditorView!
+    @IBOutlet var htmlTextView: UITextView!
+    var isTextColor = true
+    @IBOutlet weak var editButton: UIBarButtonItem!
+  
+    lazy var toolbar: RichEditorToolbar = {
+        let toolbar = RichEditorToolbar(frame: CGRect(x: 0, y: 0, width: self.view.bounds.width, height: 44))
+//        let options: [RichEditorDefaultOption] = [
+//            .bold, .italic, .underline,
+//            .unorderedList, .orderedList,
+//            .indent, .outdent,
+//            .textColor, .textBackgroundColor,
+//            .undo, .redo,
+//        ]
+        toolbar.options = RichEditorDefaultOption.all
+        return toolbar
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        editorView.delegate = self
+        editorView.inputAccessoryView = toolbar
+        editorView.editingEnabled = false
+        editorView.placeholder = "Edit here"
+        // disable scroll to fix issue with auto-scrolling when keyboard appears
+        editorView.isScrollEnabled = false
+        let html = "<b>Jesus Saves!.</b> No words of praise, no promised land to take you to-- there is no other WAY! <a href='https://slayer.net'>perfectGod.com</a>"
```

Comment:
It'd be better to totally remove this `html` instead of "masking" it, if you didn't like what it originally said :)

Reply:
I totally forgot that was in there! I guess it was a good test for how deep the review would go 😆 

I'll update that in a moment, I suppose we can't all be Slayer fans so removal makes sense 👍

Reply:
Fair enough. 

Also, in light of:
> no promised land to take you to

I'd love to know what you mean by that statement, if you're willing to discuss or defend it DM me `@besswisdom`. I see you prefer Threads, but I'm not on there.

Reply:
@cbess I pushed an quick update swapping to the less controversial Quick Brown Fox boiler plate. Hoping that is amenable for the pr review, and thanks again.

Reply:
> I'd love to know what you mean by that statement, if you're willing to discuss or defend it DM me `@besswisdom`. I see you prefer Threads, but I'm not on there.

Oh, they are the lyrics to a song by Slayer. I think I was testing something while running the Example project and changed the text to make sure I was looking at the right version of the view controller, but tried to keep it religious. That lyric floated across my mind so I used it,  and then I forgot about it, sorry for the mixup!